homework-1/src/Point.ts

```diff
@@ -0,0 +1,34 @@
+export class Point {
+    public x: number;
+    public y: number;
+
+    constructor();
+    constructor(x?: number, y?: number) {
```

Comment:
You can use newer notation like constructor(public x = 0, public y = 0) {}. In this case you don't need explicitly declare x, y.

Reply:
Fixed. Thanks a lot, didn't know about this useful feature.